app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title do "Forgot Password" end %>
+
+<% content_for :alerts do render "application/alerts" end %>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post, class: "form-signin" }) do |f| %>
+  <h2 class="form-signin-heading">Forgot your password?</h2>
+  <p class="centered-text">Send me reset password instructions</p>
+
+  <%= yield :alerts %>
```

Comment:
The parent never uses :alerts other than to determine if it is defined.

Alerts is defined on line 3 and is rendered in the devise template on line 9 using the yield. Inside the base template, the base template sees that `:alerts` has been defined, so it does not load `render "application\alerts"`.

I'd like to display the alerts in different spots depending on the part of the app. Alerts are displayed inside the login/forgot password box when using the devise layout. They are displayed right about the content when using the application layout.

The base layout displays alerts, but not through the `:alerts` symbol and only if `:alerts` has not been defined anywhere else.

Sorry if this is confusing. Please let me know if you have additional questions and if we need to discuss this. Thanks.